benches/benches/contract_root.rs

```diff
@@ -0,0 +1,48 @@
+use criterion::{
+    Criterion,
+    Throughput,
+};
+use fuel_core_types::fuel_tx::Contract;
+use rand::{
+    rngs::StdRng,
+    Rng,
+    SeedableRng,
+};
+use std::iter::successors;
+
+fn random_bytes<R: Rng + ?Sized>(n: usize, rng: &mut R) -> Vec<u8> {
+    let mut bytes = vec![0; n];
+    for chunk in bytes.chunks_mut(32) {
+        rng.fill(chunk);
+    }
+    bytes
+}
+
+pub fn contract_root(c: &mut Criterion) {
```

Comment:
Maybe you want to move this file into `vm_set` module=)

Reply:
I don't think I would like to do put this under `vm_set`, since this doesn't really touch the VM. I think it would be better to consolidate these files by adding a parent folder to contain both VM and non-VM items used for gas pricing. I will merge this PR and look at that for the next one.